### PR TITLE
Fix travis ci broken.

### PR DESCRIPTION
**Pull request by dkinzer**

### CODE REVIEW

File: solr/sample_solr_documents.yml [hunk 1]
```diff
@@ -1,2692 +0,0 @@
----
-- lc_1letter_facet:
```
Comment (mejackreed) [inline solr/sample_solr_documents.yml]: Are removing these affecting the build? I found it sometimes helpful in the past to be able to have these handy.

Comment (dkinzer) [reply solr/sample_solr_documents.yml]: I don't think so.  I did this because blacklight removed them from their repo, but I can add them back.

**Comment by dkinzer** [comment]
@mejackreed I added that file back, but note that it does not get used by any of the rake tasks.  Both `blacklight:solr:seed` and `blacklight_range_limit:seed` use the files in `spec/fixtures`.  So users would have to know those files exist and ingest them manually.

**Comment by mejackreed** [comment]
Thanks @dkinzer 👍

**Pull request merged**